client/dogs/FormPage.js

```diff
@@ -400,6 +408,17 @@ export default class IndexPage extends React.Component {
     ev.preventDefault();
   };
 
+  onDropRabies = (file: Array<Object>) => {
+    this.setState({
+      rabiesCert: file,
```

Comment:
You'll want to update your naming for the state keys to be very very accurate. So, since this is an array of files, you want the name to be something like `rabiesCertFiles`.

That being said, double-check if you want this to be an array or a single file. Your input will always be an array of one, since you've turned off `multiple` in the configuration.